src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -167,26 +189,28 @@ const Jazz: React.FC<JazzProps> = ({ setUserList, venue }) => {
         />
       )}
 
-      <div className="music-bar-content">
-        <div className="video-container">
+      <div className={musicBarContainerClasses}>
+        <div className={videoContainerClasses}>
           {!venue.hideVideo && (
             <>
-              <div className="iframe-container">
-                {venue.iframeUrl ? (
-                  <iframe
-                    key="main-event"
-                    title="main event"
-                    className="iframe-video"
-                    src={`${venue.iframeUrl}?autoplay=1`}
-                    frameBorder="0"
-                    allow={IFRAME_ALLOW}
-                  />
-                ) : (
-                  <div className="iframe-video">
-                    Embedded Video URL not yet set up
-                  </div>
-                )}
-              </div>
+              {
```

Comment:
why was that added? is anything changed here?

Reply:
it's not clear to me what this comment is about? could you expand on it?